.gitignore

```
@@ -1,4 +1,6 @@
 .idea/*
 *.iml
+*.log
 .target/*
 .src/main/resources/log/*
+/src/main/resources/log/
```

@Valdes2 Від корня прям? Може краще так?
```suggestion
src/main/resources/log/
```

---

fixed